recipe/meta.yaml

```diff
@@ -22,17 +21,19 @@ requirements:
     - pip
     - pytest
     - python
+    - setuptools
   run:
     - amqp >=5.1.1,<6.0.0
-    - backports.zoneinfo >=0.2.1    # [py<39 or python_impl == "pypy"]
-    - python-tzdata  # [py<39 or python_impl == "pypy"]
+    - backports.zoneinfo >=0.2.1    # [py<39]
```

Comment:
We don't even consider python 3.8 anymore. So python min is 3.9 and this this selector is always false and this line is never included.

Reply:
Ah yes. For my future reference "python_min" is defined in CFEP-25 and https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/f341c8e09b3940bff3751f46537bd67c3a41d38b/recipe/conda_build_config.yaml#L848